sys-devel/automake/automake-1.15.1-r1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils versionator
+
+case ${PV} in
```

Comment:
I've already said this for the kodi ebuilds: in general prefer an if over a case dispatch. Gentoo devs brains are hardwired to recognise those if blocks...